erpnext/translations/fr.csv

```diff
@@ -285,7 +285,7 @@ Asset scrapped via Journal Entry {0},Actif mis au rebut via Écriture de Journal
 "Asset {0} cannot be scrapped, as it is already {1}","L'actif {0} ne peut pas être mis au rebut, car il est déjà {1}",
 Asset {0} does not belong to company {1},L'actif {0} ne fait pas partie à la société {1},
 Asset {0} must be submitted,L'actif {0} doit être soumis,
-Assets,Les atouts,
+Assets,Actifs - Immo.,
```

Comment:
@FHenry wouldn't this mean "real estate assets" ? Assets is generic for any assets

Reply:
@marination In french Asset("Actifs") is globalize on accountancy meaning with "Immobilisation" (french word): Assets that need a depreciation over the years in accounting balance.
The lonely world "Actifs" is may be not explicit enough, it can refer to "Actives people" (like employees) and that not the purpose of this module.
Other ERP in french prefer the word "Immobilisation" for assets, but I prefer to let the two words into ERPNExt because on accountancy plan "Actifs" is the parent level of "Immobilisation"
For exemple :
https://www.odoo.com/documentation/14.0/fr/applications/finance/accounting/payables/supplier_bills/assets.html (Actifs [non courants] et Immobilisations [corporelles])
https://www.sage.com/fr-fr/logiciel-immobilisations/